lib/tpm2_policy.c

```diff
@@ -392,6 +392,89 @@ bool tpm2_policy_build_policycommandcode(ESYS_CONTEXT *ectx,
     return true;
 }
 
+bool tpm2_policy_build_policylocality(ESYS_CONTEXT *ectx,
+    tpm2_session *session, uint8_t locality) {
+
+    ESYS_TR handle = tpm2_session_get_handle(session);
+
+    TPM2_RC rval = Esys_PolicyLocality(ectx, handle,
```

Comment:
use type TSS2_RC

Reply:
I have found similar issues under lib/
https://github.com/tpm2-software/tpm2-tools/blob/2a5a90c34913ee7ae53f4f6987f08a978a2eb9be/lib/tpm2_nv_util.h#L111
https://github.com/tpm2-software/tpm2-tools/blob/2a5a90c34913ee7ae53f4f6987f08a978a2eb9be/lib/tpm2_session.c#L131

......

perhaps we should check and fix after this pr is closed

Reply:
Yeah theirs is definitely issues of this being wrong existing, just trying not to add new ones.